src/main/java/de/tum/in/www1/artemis/domain/notification/GroupNotificationFactory.java

```diff
@@ -72,49 +80,67 @@
             String notificationText) {
         String title;
         String text;
+        boolean textIsPlaceholder;
+        String[] placeholderValues;
         NotificationPriority priority = MEDIUM;
 
         switch (notificationType) {
             case EXERCISE_RELEASED -> {
```

Comment:
I think all exercise related notification should contain the course title.

Reply:
I agree

Reply:
I added it.